kubernetes/customresourcedefinitions.gen.yaml

```
@@ -413,6 +413,28 @@ spec:
                                     given time.
                                   format: int32
                                   type: integer
+                                retryBudget:
+                                  description: The amount of concurrent retries allowed
+                                    in the case of failure, a percentage of the active
+                                    requests.
+                                  properties:
+                                    budgetPercent:
+                                      description: The limit on concurrent retries
+                                        as a percentage of active requests and active
+                                        pending requests.
+                                      properties:
+                                        value:
+                                          format: double
+                                          type: number
```

Why not add sane limits for percentage variable?
```suggestion
                                          type: number
                                          minimum: 0
                                          maximum: 100
```